### PR TITLE
Adapted and fixed some issues for ansible 2.0

### PR DESCRIPTION
**Pull request by madonius**
Is tested in Ansible 2.0.1 and works as expected. Testing in another environment would be great.

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
   poll: 0
   ignore_errors: true
-  sudo: yes
+  become: true
 
 - name: Wait for restart to finish
-  local_action: wait_for host={{ inventory_hostname }}
-                state=started
-  sudo: no
+  local_action: wait_for host={{ ansible_ssh_host }} state=started port=22 delay=50 timeout=120
```
Comment (charleskorn) [inline handlers/main.yml:10]: Is there a reason why you explicitly set the port here? If so, would it be better to use the `ansible_ssh_port` variable perhaps?

Also, apparently `ansible_host` is preferable over `ansible_ssh_host` in Ansible 2.0 (see the note box under http://docs.ansible.com/ansible/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable)

**Comment by charleskorn** [comment]
Thanks for your pull request @madonius!

File: handlers/main.yml (hunk 1 above)
Comment (madonius) [inline handlers/main.yml:10]: You are right, I did not know about the ansible_ssh_port and the ansible_host thing merely sliped past me. Thanks.

Comment (charleskorn) [inline handlers/main.yml:10]: No worries... would you be able to update the PR? Then I'll merge it.

**Pull request closed without merging**